src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.css

```diff
@@ -0,0 +1,48 @@
+.THIS.slds-tabs_card {
+    background: white;
+}
+/*formatting for edit mode checkboxes*/
+.THIS .templateInputCheckbox{
+    display: inline-flex;
+}
+/*give info icons extra space*/
+.THIS .slds-form-element__icon{
+    margin-left: .2rem;
+}
+/*padding for helptext icon*/
+.THIS .uiOutputCheckbox {
+    padding-right: .5rem;
+}
+.THIS.header {
+    margin-bottom: .75rem;
+}
+.THIS .headerButtonActions {
+    padding-top: .75rem;
+}
+.THIS .headerText {
+    padding-top: .5rem;
+}
+.THIS .slds-accordion__summary-heading lightning-primitive-icon {
+    padding-right: .5rem;
+}
+/*override different sized list options*/
+.THIS .wideListbox .slds-dueling-list__column_responsive .slds-dueling-list__options {
+    min-width: 240px;
+}
+/*removes the extra label and automatic tooltip for inputFields*/
```

Comment:
if this is about hiding standard Data Import help text and labels, make a note of why we're doing that here